addons/website_blog/static/src/snippets/s_latest_posts/000.js

```diff
@@ -16,10 +16,16 @@ publicWidget.registry.js_get_posts = publicWidget.Widget.extend({
      */
     start: function () {
         var self = this;
-        var limit = self.$target.data('postsLimit') || 3;
-        var blogID = self.$target.data('filterByBlogId');
-        var template = self.$target.data('template') || 'website_blog.s_latest_posts_list_template';
-        var loading = self.$target.data('loading');
+        const data = self.$target[0].dataset;
+        const limit = parseInt(data.postsLimit) || 4;
+        const blogID = parseInt(data.filterByBlogId);
+        // Compatibility with old template xml id
+        if (data.template === 'website_blog.s_latest_posts_big_orizontal_template') {
```

Comment:
@sdegueldre I changed to check that the original name *endsWith* the old xml id part (since it was not in website_blog before).